editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -734,7 +734,7 @@ void TileSetEditor::_on_edit_mode_changed(int p_edit_mode) {
 			tools[SHAPE_KEEP_INSIDE_TILE]->hide();
 
 			tools[TOOL_SELECT]->set_pressed(true);
-			tools[TOOL_SELECT]->set_tooltip(TTR("LMB: Set bit on.\nRMB: Set bit off.\nClick on another Tile to edit it."));
+			tools[TOOL_SELECT]->set_tooltip(TTR("LMB: Set bit on.\nRMB: Set bit off.\nSHIFT + LMB: Set bit to ignored\nClick on another Tile to edit it."));
```

Comment:
End the added sentence with a dot, like other ones. And use `Shift+LMB` (not full caps, no space).

Reply:
Suggestion: what about change `Set bit to ignored` to `Set bit to whatever`? because I think `ignored` is not very clear.

Reply:
Done